pwndbg/commands/version.py

```diff
@@ -82,10 +85,13 @@ def version():
 bugreport_parser.add_argument(
     "--run-browser", "-b", action="store_true", help="Open browser on github/issues/new"
 )
+bugreport_parser.add_argument(
```

Comment:
These two options are going to be mutually exclusive right? We should probably use `add_mutually_exclusive_group` : https://stackoverflow.com/questions/17909294/python-argparse-mutual-exclusive-group